src/components/ImageCard/_image-hotspot-content.scss

```
@@ -0,0 +1,11 @@
+@import '../../globals/vars';
+
+.#{$iot-prefix}--hotspot-content h4 {
+  white-space: nowrap;
+  overflow-x: hidden;
```

should combine these. 
```suggestion
  overflow: hidden;
```